templates/stamp/node-jumpbox.json

```diff
@@ -0,0 +1,168 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "apiSettings": {
+            "type": "object",
+            "metadata": {
+                "Description": "API version of the resource type"
+            }
+        },
+
+        "networkSettings": {
+            "type": "object",
```

Comment:
Like customSettings, shouldn't networkSettings be secureObject since it contains key/password data?